front/app/components/EsriMap/utils.tsx

```diff
@@ -1,14 +1,41 @@
 import { colors } from '@citizenlab/cl2-component-library';
 
+// components
+import VectorTileLayer from '@arcgis/core/layers/VectorTileLayer';
+import WebTileLayer from '@arcgis/core/layers/WebTileLayer';
+import Layer from '@arcgis/core/layers/Layer';
+
 // constants
-import { BASEMAP_AT_ATTRIBUTION, MAPTILER_ATTRIBUTION } from './constants';
+import {
+  BASEMAP_AT_ATTRIBUTION,
+  DEFAULT_TILE_PROVIDER,
+  MAPTILER_ATTRIBUTION,
+} from './constants';
 
 // components
 import SimpleMarkerSymbol from '@arcgis/core/symbols/SimpleMarkerSymbol';
 
+// getDefaultBasemap
+// Description: Gets the correct basemap given a certain tileProvider URL.
+export const getDefaultBasemap = (tileProvider: string | undefined): Layer => {
+  if (tileProvider?.includes('wien.gv.at/basemap')) {
+    return new VectorTileLayer({
+      // For Vienna's custom basemap, we fetch a vector tile layer
+      portalItem: {
+        id: 'd607c5c98e6a4e1fbd3569e38c5c8a0c',
+      },
+    });
+  } else {
+    return new WebTileLayer({
+      urlTemplate: tileProvider || DEFAULT_TILE_PROVIDER,
+      copyright: getTileAttribution(tileProvider || ''),
+    });
+  }
+};
+
```

Comment:
If the tileProvider is Vienna's custom one, we want to fetch the basemap as a VectorTileLayer hosted on Esri. Otherwise, we fetch the basemap using the tileProviderURL from AdminHQ.